### PR TITLE
Setup hello microverse

### PR DESCRIPTION
**Pull request by gergitokusse**
### The following concepts were done on the project  as per requested/commented by code reviewer

- Old pull request title was changed to a descriptive title
- feature branch name changed to a descriptive name

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,283 @@
-# Hello-World_project
\ No newline at end of file
+<a name="readme-top"></a>
```
Comment (topeogunleye) [inline README.md:1]: - [ ] As suggested by the previous reviewer, please kindly personalize the template to match your project. See an example of a customized Hello Mocroverve Project compared to yours:

In the case of README consider not leaving any of the sections in the [Microverse README](https://github.com/microverseinc/readme-template) template as it's a project requirement.

|   |  |
| ------------- | ------------- |
| Your README  | image)  |
|  Professional README  | Recording 2023-02-14 at 12 52 20 (5)  |

File: README.md [hunk 2]
```diff
@@ -1 +1,283 @@
...
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
```
Comment (topeogunleye) [inline README.md]: [ OPTIONAL ] Please consider removing this instructional text from your readme file, this text serves to guide you on how to fill this section of the readme file. You are supposed to remove the instructional text after filling in this section, please remove it.

File: README.md [hunk 3]
```diff
@@ -1 +1,283 @@
...
+- **[key_feature_2]**
+- **[key_feature_3]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
```
Comment (topeogunleye) [inline README.md]: [ OPTIONAL ] Please consider removing this instructional text from your readme file, this text serves to guide you on how to fill this section of the readme file. You are supposed to remove the instructional text after filling in this section, please remove it.

File: README.md [hunk 4]
```diff
@@ -1 +1,283 @@
...
+
+- [Live Demo Link](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
```
Comment (topeogunleye) [inline README.md]: [ OPTIONAL ] Please consider removing this instructional text from your readme file, this text serves to guide you on how to fill this section of the readme file. You are supposed to remove the instructional text after filling in this section, please remove it.

File: README.md [hunk 5]
```diff
@@ -1 +1,283 @@
...
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
```
Comment (topeogunleye) [inline README.md]: [ OPTIONAL ] Please consider removing this instructional text from your readme file, this text serves to guide you on how to fill this section of the readme file. You are supposed to remove the instructional text after filling in this section, please remove it.

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @gergitokusse, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
* No linter errors. :heavy_check_mark:
* Followed Github flow :heavy_check_mark:
* PR has a good title and summary :heavy_check_mark:

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

You can also consider:
- [ ]  Next time, and for future commits, I kindly suggest that you please make use of imperative mood in the subject line for your commit messages.
:x: Bad Commit Messages :x:
image

### ✔️ Good Commit Messages: ✔️
✔️  Add fix for dark mode toggle state
✔️  Update README file.
✔️  Add styles to h1.

- [ ] Please refer to this article: please read it and apply it in your work from now on: [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md);

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by victorokoroji — APPROVED** [review]
Hi @gergitokusse, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._###

**Pull request merged**